libraryapi/pergamum.py

```diff
@@ -13,14 +13,16 @@
 
 
 class DadosMarc(BaseModel):
-    """Represents a Dados_marc object received from Pergamum webservice"""
+    """Represents a Dados_marc object received from Pergamum Web Service"""
 
     paragrafo: list[str]
     indicador: list[Optional[str]]
     descricao: list[Optional[str]]
 
 
-class Pergamum:
+class PergamumWSrequest:
+    """Represents a connection and a request to the Pergamum Web Service"""
```

Comment:
Handle conections and requests to Pergamum Web Services, keeping a pool of connections with some new Pergamum Web Service requested.